face-kiosk/face-kiosk-application/src/main/java/com/pillar/skynet/facekiosk/application/Main.java

```diff
@@ -43,11 +50,21 @@ public static void main(String[] s) throws IOException, FrameGrabber.Exception {
 			LOG.info("Found a face tho.");
 		}
 
+		CvRect r = new CvRect(cvGetSeqElem(faces, 0));
```

Comment:
The '0' here is the index in the faces CvSeq (think of it as an array of stuff) for the face to pick out. This is where we'll iterate through and repeat this process if there are multiple faces in frame.